controllers/argocd/notifications.go

```diff
@@ -413,6 +524,547 @@ func getNotificationsResources(cr *argoprojv1a1.ArgoCD) corev1.ResourceRequireme
 	return resources
 }
 
+// getDefaultNotificationsConfig returns a map that contains default triggers and template configurations for argocd-notifications-cm
+func getDefaultNotificationsConfig() map[string]string {
```

Comment:
Just wondering if shifting this to a different file would help keep file easily readable for future. The file just looks too big to me especially as we are planning to refactor the codebase into smaller chunks.

Reply:
Yeah, this should go into a separate `notifications_util.go` or something 
but I thought that would be better suited to the refactoring task - I didn't want to conflate the 2 efforts in a single PR

Reply:
done